earth_enterprise/src/fusion/autoingest/plugins.src/Database.src

```diff
@@ -881,8 +885,12 @@ DatabaseAssetVersionImplD::CreateSubsampledTerrain(
       FindNeededTerrainInsets(ssGenInfo[i].coverage,
                               terrainInsetPtrs, numTerrain,
                               neededIndexes,
-                              beginMinifyLevel,
-                              endMinifyLevel);
+                              (MiscConfig::Instance().UseMinification
```

Comment:
Unless I'm missing something it doesn't look like you need these changes. Since you set begin/endMinifyLevel correctly above, you shouldn't need to check the flag again here.